package.json

```diff
@@ -4,17 +4,23 @@
   "description": "Website for decoding CIDs",
   "main": "index.js",
   "scripts": {
-    "build": "parcel build index.html --no-minify --public-url '.'",
+    "clean": "rm -rf dist/*",
+    "build": "parcel build index.html --public-url '.'",
     "dev": "parcel index.html",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "deploy": "ipfs add -r -q dist | tail -n1 >versions/current; cat versions/current >> versions/history; cat versions/current"
```

Comment:
Since with jenkins we're doing versioning differently, we can change this script to just build the site + add to IPFS but skip everything regarding having the versions saved in scm.

Reply:
I'll strip it back to `ipfs add -r -q dist` and remove the versions folder.